build.ps1

```diff
@@ -31,9 +31,31 @@ function conan_create_profile($profile) {
   $build_type = $profile -replace "default_"
   $profile_path = "$conan_dir/profiles/$profile"
 
+  if (!$Env:Qt5_DIR) {
+    Write-Host ""
+    Write-Host "Compile Qt from source or use official distribution?"
+    Write-Host "====================================================`r`n"
+    Write-Host "Orbit depends on the Qt framework which can be either automatically compiled from source (takes a long time),"
```

Comment:
nit. I think we should emphasize that building Qt from source could take *hours*.

"...compiled from source (can take hours)," or something along those lines?

Reply:
Yes, makes sense! Done!